llvm/lib/SYCLLowerIR/GlobalOffset.cpp

```diff
@@ -59,11 +59,21 @@ ModulePass *llvm::createGlobalOffsetPassLegacy() {
   return new GlobalOffsetLegacy();
 }
 
+void getLoads(Instruction *P, SmallVectorImpl<Instruction *> &Traversed,
```

Comment:
This function could use a short comment to clarify what it does and returns through the references

Reply:
I would prefer if `getLoads` was either `static`or lived in an anonymous namespace, so the symbol doesn't leak.
+1 for comments as well.

Reply:
I've turned it into a static function and also added a comment